api/lib/JSONValidator.js

```diff
@@ -0,0 +1,26 @@
+const Ajv = require('ajv');
+const ajv = new Ajv({allErrors: true});
+
+const Schemas = require('./schemas');
+const errorMessages = require('./error-messages');
+
+function JSON(){
```

Comment:
It's a good practice to have class name to be the same as the file name, also, JSON is a [default object from js](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON), so this may conflict with it someway in the future, I would suggest changing this to JSONValidator

Reply:
Ok I'm going to change it :+1: